deployments/crds/hephaestus.dominodatalab.com_imagebuilds.yaml

```diff
@@ -3,8 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.3
-  creationTimestamp: null
```

Comment:
Do we know why this was here?

Reply:
its all code generation.
guess it was a think in the older code gen but not anymore.